test/evergreen.yml

```diff
@@ -874,6 +881,21 @@ tasks:
 
             ${test_env_vars|} $(pwd)/../test/csuite/timestamp_abort/smoke.sh 2>&1
 
+  - name: csuite-timestamp-abort-stress-test
+    tags: ["pull_request"]
+    depends_on:
+      - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+
+            ${test_env_vars|} $(pwd)/../test/csuite/timestamp_abort/smoke.sh -s 2>&1
```

Comment:
How long does the smoke.sh typically take to run?

Reply:
Between 5 and 6 minutes.